src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.ts

```diff
@@ -142,6 +144,7 @@ export const useLensSuggestions = ({
     currentSuggestion: histogramSuggestion ?? currentSuggestion,
     suggestionUnsupported: !currentSuggestion && !histogramSuggestion && isPlainRecord,
     isOnHistogramMode: Boolean(histogramSuggestion),
+    histogramQuery: histogramQuery.current ? { esql: histogramQuery.current } : undefined,
   };
```

Comment:
ℹ️ This is done to communicate correctly the histogram query (as it is not the same that Discover uses aka the unified search query)